client/visitor.go

```diff
@@ -377,12 +377,24 @@ func (sv *SUDPVisitor) Run() (err error) {
 func (sv *SUDPVisitor) dispatcher() {
 	xl := xlog.FromContextSafe(sv.ctx)
 
+	var (
+		visitorConn net.Conn
+		err         error
+	)
+
 	for {
 		// loop for get frpc to frps tcp conn
```

Comment:
`// Create a new work connection between frpc and frps when receiving new contents and there is no active work connection`